tests/acl/null_route/test_null_route_helper.py

```diff
@@ -68,35 +69,48 @@
 
 
 @pytest.fixture(scope="module", autouse=True)
-def remove_dataacl_table(rand_selected_dut):
+def remove_data_everflow_acl_table(duthosts):
     """
-    Remove DATAACL to free TCAM resources
+    Remove DATAACL and EVERFLOW to free TCAM resources.
+    The change is written to configdb as we don't want DATAACL recovered after reboot
     """
-    TABLE_NAME = "DATAACL"
-    lines = rand_selected_dut.shell(cmd="show acl table {}".format(TABLE_NAME))['stdout_lines']
-    data_acl_existing = False
-    for line in lines:
-        if TABLE_NAME in line:
-            data_acl_existing = True
-            break
-    if not data_acl_existing:
-        yield
-        return
-    # Remove DATAACL
-    logger.info("Removing ACL table {}".format(TABLE_NAME))
-    rand_selected_dut.shell(cmd="config acl remove table {}".format(TABLE_NAME))
+    TABLE_NAME_1 = "DATAACL"
+    TABLE_NAME_2 = "EVERFLOW"
+    for duthost in duthosts:
+        lines = duthost.shell(cmd="show acl table {}".format(TABLE_NAME_1))['stdout_lines']
+        data_acl_existing = False
+        for line in lines:
+            if TABLE_NAME_1 in line:
+                data_acl_existing = True
+                break
+        if data_acl_existing:
+            # Remove DATAACL
+            logger.info("Removing ACL table {}".format(TABLE_NAME_1))
+            cmds = [
+                "config acl remove table {}".format(TABLE_NAME_1),
+                "config save -y"
+            ]
+            duthost.shell_cmds(cmds=cmds)
+
+        lines = duthost.shell(cmd="show acl table {}".format(TABLE_NAME_2))['stdout_lines']
+        everflow_acl_existing = False
+        for line in lines:
+            if TABLE_NAME_2 in line:
+                everflow_acl_existing = True
+                break
+        if everflow_acl_existing:
+            # Remove DATAACL
+            logger.info("Removing ACL table {}".format(TABLE_NAME_2))
+            cmds = [
+                "config acl remove table {}".format(TABLE_NAME_2),
+                "config save -y"
+            ]
+            duthost.shell_cmds(cmds=cmds)
+
     yield
-    # Recover DATAACL
-    config_db_json = "/etc/sonic/config_db.json"
-    output = rand_selected_dut.shell("sonic-cfggen -j {} --var-json \"ACL_TABLE\"".format(config_db_json))['stdout']
-    try:
-        entry = json.loads(output)[TABLE_NAME]
-        cmd_create_table = "config acl add table {} {} -p {} -s {}"\
-            .format(TABLE_NAME, entry['type'], ",".join(entry['ports']), entry['stage'])
-        logger.info("Restoring ACL table {}".format(TABLE_NAME))
-        rand_selected_dut.shell(cmd_create_table)
-    except Exception as e:
-        pytest.fail(str(e))
+    # Recover DUT by reloading minigraph
+    for duthost in duthosts:
+        config_reload(duthost, config_source="minigraph")
```

Comment:
1. Config reload takes time. Is there a faster way to restore the removed ACL tables?
2. If config reload is the only way to restore configuration, please use `config_source="running_golden_config"`.

Reply:
Thanks for your comments, i have removed config reload and used other method to reconfigure the acl.